docs/Makefile

```diff
@@ -20,3 +20,27 @@ docs-up-docker:
 		-p 4000:4000 \
 		-v $(PWD):/srv/jekyll \
 		jekyll/jekyll:3.8 jekyll serve --livereload
+
+# This commands is helpful when you want to convert image to the webp format
+# All images quality set to 20% but it just reduce the size of image, but not the display quality
+
+# To run image convertor just run next command in the terminal
+#
+# PATH_TO_IMAGES=path_to_the_images IMAGES_EXTENSION=png make docs-change-images
+#
+# This command will reduce all images with extionsion and folder defined in the command.
+# All images with defined extensions will be removed.
+# After that it is nesessary to update src attribute for image tag
+
+docs-images-to-webp:
+	docker run -it --rm -v \
+		$(PWD)/$(PATH_TO_IMAGES):/data \
+		elswork/cwebp \
+		find ./ -type f -name '*.$(IMAGES_EXTENSION)' -exec sh -c 'cwebp -q 20 -m 6 $$1 -o "$${1%.png}.webp"' _ {} \;
```

Comment:
quality 20 seems like too low, I clearly see image quality degradation. We need to remember that these images can be re-used by VM users. Would be pity if the images will have that low quality.

Reply:
Hi @hagen1778 ! Can you show what images have degradation I will check what produce it degradation, or maybe recreate them

Reply:
I've compared two following images:
* https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/9149353a364b3fae49a82d4740a1c3946e0c4569/docs/Cluster-VictoriaMetrics_cluster-scheme.png
* https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/e1eb0c5d06bb1c86efb0a7b3aa0be6b7542c767a/docs/Cluster-VictoriaMetrics_cluster-scheme.webp

I find text on images to have obscure artifacts.

Reply:
used quality 40% it looks like better